### PR TITLE
Nhu - Feature 6: Delete a Movie

### PR DESCRIPTION
**Pull request by nhunpn**
I finished the delete a movie function and the tests for the delete function.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -55,7 +55,11 @@ def update_movie(movie_id: int):
     return redirect(f'/movies/{movie_id}')
 
 
+# Nhu's delete movie function
 @app.post('/movies/<int:movie_id>/delete')
 def delete_movie(movie_id: int):
-    # TODO: Feature 6
-    pass
```
Comment (AnessaLeng) [inline app.py]: probably remove this

File: tests/unit/test_delete_movie.py
Comment (AnessaLeng) [inline tests/unit/test_delete_movie.py:1]: Looks good!

File: tests/e2e/test_delete_movie_page.py
Comment (AnessaLeng) [inline tests/e2e/test_delete_movie_page.py:1]: Looks good!

File: app.py [hunk 2]
```diff
@@ -55,7 +55,11 @@ def update_movie(movie_id: int):
...
+    # Feature 6
+    for movie in movies:
+        if movie['movie_id'] == movie_id:
+            movies.remove(movie)
+    return redirect('/movies')
```
Comment (AnessaLeng) [inline app.py]: add movies=movies in the redirect, so redirect('/movies', movies=movies) so that it matches with the list all movies template

**Review by AnessaLeng — APPROVED** [review]
Everything looks good!

**Review by jaidenDaya — APPROVED** [review]
Looks good!

**Review by jaidenDaya — APPROVED** [review]
Looks good!

**Pull request merged**